src/views/v2/list/top-bar/datepicker.php

```diff
@@ -46,7 +46,7 @@ class="tribe-events-c-top-bar__datepicker-time"
 			</time>
 		<?php endif; ?>
 		<?php if ( $show_end ) : ?>
-			<span class="tribe-events-c-top-bar__datepicker-separator">&mdash;</span>
+			<span class="tribe-events-c-top-bar__datepicker-separator"> - </span>
```

Comment:
this should be the separator var

Reply:
I didn't change that and wanted to ask design/strategy, because technically the option for the separator is for time range and not the datepicker.

Reply:
https://tribe.slack.com/archives/CEUHNUF3R/p1573755087278900?thread_ts=1573753404.276600&cid=CEUHNUF3R - We should keep the hyphen for the datepicker and use the option for dates.